scripts/make-pr.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -euo pipefail
+
+repo=${REPO:-mongodb/mongodb-atlas-kubernetes}
+github_token=${GITHUB_TOKEN:-unset}
+
+if [[ $(git diff --stat) = '' ]]; then
+  echo 'No PR needed, git is clean'
+  exit 0
+fi
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+update_timestamp=$(date -u --iso-8601='minutes')
+title="AKOBot Automatic Update ${update_timestamp}"
+
+echo "Touching pkg/controller/touch.go to force the full test suite to run"
+echo "package controller // updated at: ${update_timestamp}" \
+  > "${SCRIPT_DIR}/../pkg/controller/touch.go"
+
+echo "Creating branch akobot-update and pushing it"
+git branch -m akobot-update
+git add .
+git commit -m "${title}"
+git push -fu origin akobot-update
+
+echo "Creating PR"
+curl -L \
+  -X POST \
+  -H "Accept: application/vnd.github+json" \
+  -H "Authorization: Bearer ${github_token}" \
+  -H "X-GitHub-Api-Version: 2022-11-28" \
+  "https://api.github.com/repos/${repo}/pulls" -d "{\"title\":\"${title}\"}"
```

Comment:
Suggestion: You can use Github CLI and make it better to manage

Reply:
If we add `gh` we need to manage its installation in Makefile as well, `curl` comes preinstalled in CI and almost everywhere, and it is easier to install. I would be happy to change my mind if `gh` can be easily installable with `go get ...` but that did not work for me.

I can look into it, but for just one endpoint, depending on `gh` might be an overkill.

Reply:
With GH cli we could not only make it more elegant(IMO) but also add label(cloud-tests) to the PR